components/03-modules/slider/slider.hbs

```diff
@@ -1,37 +1,8 @@
 <div class="slider {{ class }}" {{{ attributes }}}>
     <div class="slider__handler">
         <div class="slider__slides">
-            {{#each products }}
-                <div class="slider__item">
-                    <a href="{{ this.link }}"
-                       class="slider__link"
-                       aria-label="{{ this.ariaLabel }}"
-                    >
-                        {{ render '@image' this.image }}
-
-                        <div class="slider__item-handler">
-                            {{ render '@heading' this.heading }}
-
-                            <div class="slider__price">
-                                {{#if this.oldPrice}}
-                                    <span class="slider__price-old">
-                                        {{ this.oldPrice }}
-                                    </span>
-
-                                    <span class="slider__price-new">
-                                        {{ this.price }}
-                                    </span>
-                                {{else}}
-                                    {{ this.price }}
-                                {{/if}}
-                            </div>
-                        </div>
-
-                        {{#if badge}}
-                            {{ render '@badge' this.badge }}
-                        {{/if}}
-                    </a>
-                </div>
+            {{#each elements }}
+                {{ render (component content) contentContext }}
             {{/each}}
         </div>
     </div>
```

Comment:
code looks ok for me but the slider on wider screen doesn't work properly, I see mutliple slides and I cannot see the arrows. It should work like that? we should be able to set how many slides is visible? and there are some option to media query? I try also with images as content. Could you please recheck this module with multiple slides elements?

Reply:
ok, for now, I've added new div which represents one item in the slider. We can add class to it and modify as we want with size. Navigation in the slider, for now, can have own class and we can delete hiding arrow on the desktop (b/c in the project are hidden in desktop). For the preview, I've added images elements. And to manage configu of slider we can pass attributes as `data` and give there props as in docs. You can see in the preview.